voxblox_ros/include/voxblox_ros/ros_params.h

```diff
@@ -119,7 +119,7 @@ inline EsdfIntegrator::Config getEsdfIntegratorConfigFromRosParam(
       getTsdfIntegratorConfigFromRosParam(nh_private);
 
   esdf_integrator_config.min_distance_m =
-      tsdf_integrator_config.default_truncation_distance;
+      tsdf_integrator_config.default_truncation_distance / 2.0;
```

Comment:
is this needed or just a better default?

Reply:
Better default in my opinion (and in the paper's quantitative evaluation).